content/compute-at-sdsc.md

```diff
@@ -0,0 +1,68 @@
+Title: Bring your computing to the San Diego Supercomputer Center
+Date: 2018-10-24 18:00
+Author: Andrea Zonca
+Tags: comet, sdsc, xsede
+Slug: compute-at-sdsc
+
+I am often asked what computing resources are available at the San Diego Supercomputer Center for scientists and what is the best way to be granted access. I decided to write a blog post with an overview of all the options, consider that I'm writing this in October 2018, so please cross-check on the official websites.
+
+## Comet
+
```

Comment:
Suggest you add a URL for the Comet user guide. http://www.sdsc.edu/support/user_guides/comet.html